Fix FTBFS with gcc 4.4 (Patch contributed by Artur Rona <ari-tczew@tlen.pl>)
--- a/src/utils.cc
+++ b/src/utils.cc
@@ -270,7 +270,7 @@
     return false;
   }
 
-  ofd = open(f2, O_WRONLY|O_CREAT|O_TRUNC);
+  ofd = open(f2, O_WRONLY|O_CREAT|O_TRUNC, 0644);
   if (ofd == -1) {
     error = std::strerror(errno);
     close(ifd);
